layers/+lang/javascript/packages.el

```diff
@@ -15,6 +15,7 @@
         coffee-mode
         company
         (company-tern :requires company)
+        eslint-fix
```

Comment:
this should be conditionally installed only if `javascript-eslint-fix-on-save`, I believe it's:

```elisp
(eslint-fix :toggle javascript-eslint-fix-on-save)
```

Then you can remove the `:if` in the use-package below.

Reply:
ah smart. thanks